### PR TITLE
Merge release/5.2.0 into develop

### PR DESCRIPTION
**Pull request by danielmain**
This PR merges the release 5.2.0 branch into develop branch

### CODE REVIEW

**Review by przemyslaw-wlodek — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -40,10 +40,16 @@ Daedalus - Cryptocurrency Wallet
 
    substituters = https://cache.iog.io https://cache.nixos.org/
    trusted-substituters =
-   trusted-public-keys = hydra.iohk.io:f/Ea+s+dFdN+3Y/G+FDgSq+a5NEWhJGzdjvKNGv0/EQ= cache.nixos.org-1:6NCHdD59X431o0gWypbMrAURkbJ16ZPMQFGspcDShjY=
+   trusted-public-keys = cache.nixos.org-1:6NCHdD59X431o0gWypbMrAURkbJ16ZPMQFGspcDShjY=
```
Comment (michalrus) [inline README.md:43]: We still need this key, unfortunately it's named `hydra.iohk.io`, but really it is for `https://cache.iog.io` 😔

**Review by michalrus — CHANGES_REQUESTED** [review]
I found just one change needed 🙏 🙏 🙏

File: README.md (hunk 1 above)
Comment (danielmain) [reply README.md]: I changed it back

**Review by michalrus — APPROVED** [review]
Thank you! =)

**Pull request merged**